zer0-os/zOS
# Channels app

### What does this do?
Adds a new app - Channels
### Why are we making this change?
Because we want some channels

### How do I test this?
in the dev console, run: `FEATURE_FLAGS.channelsApp = true;`
refresh
click 'Channels' in the app menu.
you should see a list of channels

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
